src/main/java/com/conveyal/gtfs/loader/Table.java

```diff
@@ -103,6 +103,9 @@ public class Table {
     /** Key(s) that uniquely identify an entry in the respective table.*/
     private String[] primaryKeyNames;
 
+    /** Prefixed to exported tables/files that are not part of the GTFS spec. */
+    public static final String PROPRIETARY_FILE_PREFIX = "datatools_";
```

Comment:
`datatools_` is sad that you are not showing the love! 

It's the complete pattern data not just the name. I like/love `datatools_` because it defines the origin.

Reply:
If you like it, I don't feel that strongly about it! It definitely identifies it well as a custom file that's coming out of datatools